Dockerfile.db

```
@@ -0,0 +1,22 @@
+# This Dockerfile contains the image specification of our database
+FROM postgres:15.3-alpine
+
+COPY ./dev/out/postgresdb.key /var/lib/postgresql
+COPY ./dev/out/postgresdb.crt /var/lib/postgresql
+
+COPY ./dev/out/myCA.crt /var/lib/postgresql
+COPY ./dev/out/myCA.crl /var/lib/postgresql
```

To get mine to work I had to update these to
```suggestion
COPY ./out/postgresdb.key /var/lib/postgresql
COPY ./out/postgresdb.crt /var/lib/postgresql

COPY ./out/myCA.crt /var/lib/postgresql
COPY ./out/myCA.crl /var/lib/postgresql
```
I also copied the `Dockerfile.db` to `dev/` – do we wanna decide on that folder since we already have a bash script in there? I'd be fine w/ that but we should decide together